Not linux compatible (WatchOSX)

There is no linux build for the WatchOSX import, so building goauto fails with the following error:

    watchosx.go:12:2: no buildable Go source files in src/github.com/go-fsnotify/fsevents

--- a/watchosx.go
+++ b/watchosx.go
@@ -1,5 +1,6 @@
 // Copyright 2015 Davin Hills. All rights reserved.
 // MIT license. License details can be found in the LICENSE file.
+// +build darwin
 
 package goauto
 

--- a/watchosx_other.go
+++ b/watchosx_other.go
@@ -1,0 +1,10 @@
+// Copyright 2015 Davin Hills. All rights reserved.
+// MIT license. License details can be found in the LICENSE file.
+// +build !darwin
+
+package goauto
+
+// NewWatchOSX returns a standard WatchFS, if OS is not OSX
+func NewWatchOSX() Watcher {
+	return NewWatchFS()
+}

